docs/apache-airflow/concepts/priority-weight.rst

```diff
@@ -0,0 +1,32 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+.. _concepts:priority-weight:
+
+Priority Weights
+================
+
+``priority_weight`` defines priorities in the executor queue. The default ``priority_weight`` is ``1``, and can be
+bumped to any number. When sorting the executor queue to evaluate which task should be executed next, we calculate the
+true ``priority_weight``, which is summed up with all of the ``priority_weight`` values from its downstream tasks.
+You can use this to bump a specific important task and the whole path to that task gets prioritized accordingly.
+
+The ``priority_weight`` parameter can be used in conjunction with :ref:`concepts:pool`.
+
+Tasks will be scheduled as usual while the slots fill up. Once capacity is reached, runnable tasks get queued and their
+state will show as such in the UI. As slots free up, queued tasks start running based on the
+``priority_weight`` (of the task and its descendants).
```

Comment:
This is the exact text you wrote in lines 41-44 of pools.rst.
I think this text isn't relevant here.

Reply:
Oops must have copied it by accident when refactoring to master. Will remove.